### DISCUSSION TITLE
Hue API v2 support?

### DISCUSSION THREAD

**Discussion by dr3amr** [diyhue/diyHue#903] [Ideas]
@mariusmotea Hi, great to see project is alive, not like the other HA one, Emulated HUE.
Any plan to implement Hue API v2? Currently this dyihue not working with official app. Also I would assume v2 brings new features / changes.

**Comment by mariusmotea** [comment]
Hi,

API V2 is implemented, but until it become public (and even after in some cases) this has lot of changes and it is hard to keep it up to date. What issue do you have with official Hue app. You are using an image recent from master branch? You are on iPhone or Android?

**Reply by dr3amr** [reply]
Seems the app is trying to push mandatory update to hub. I recall the mandatory update was v2 API one. And app does not allow to bypass the update so one has access to lights and other configured with hub.
![Screenshot_20230401-114232_Hue](https://user-images.githubusercontent.com/5399579/229280263-a160b981-0899-4ed4-894c-bb0e36183289.png)

**Comment by mariusmotea** [comment]
ok, let me do a quick update then with latest swversion and apiversion.

**Reply by dr3amr** [reply]
Understood it is hard to keep up to date with all firmware changes hue team does. Will use other apps available if cannot use the official app due to "mandatory" update. If it is possible / not that hard to bring diyhue up to date date that is great. It is also great you were able to implement v2 API. Thanks for your work.

**Reply by mariusmotea** [reply]
i unserstand some users put the swversion to 9999999999 and had success with that.

**Comment by mariusmotea** [comment]
https://github.com/diyhue/diyHue/commit/bff921295cdc4354b13622e693ae320b6485bec3

this must do the trick
